cardano-cli/test/cardano-cli-golden/files/golden/help/allegra_transaction.cli

```diff
@@ -35,6 +36,7 @@ Available commands:
   policyid                 Calculate the PolicyId from the monetary policy
                            script.
   calculate-min-fee        Calculate the minimum fee for a transaction.
+  calculate-min-fee-2      Calculate the minimum fee for a transaction.
```

Comment:
Can you add a description how those two commands differ?

Reply:
The `calculate-min-fee-2` is a lot more accurate because it uses `calculateMinTxFee` which wraps a ledger function `calcMinTxFee` that does the calculation in a more accurate way but it takes a `utxo` argument which `evaluateTransactionFee` did not so that's why the inputs are different.